docs/whats-new.asciidoc

```diff
@@ -13,140 +13,194 @@ Other versions: {security-guide-all}/8.5/whats-new.html[8.5] | {security-guide-a
 
-*{ml-cap} job enhancements*
+Now, when you {security-guide}/rules-ui-management.html#import-export-rules-ui[export and import] a rule that uses action connectors, you don't need to handle the connectors separately. Note that sensitive information about the connector (such as authentication credentials) is not included in the export file. You must re-add missing connector details after importing detection rules.
 
-The following enhancements simplify managing {ml} detection rules and their associated {ml} jobs:
+*Cross-cluster search support for indicator match rules*
 
-* Required {ml} jobs start automatically when you create a new {ml} rule.
-* The rule details page includes toggles to turn on {ml} jobs for existing rules.
-* The Rules page displays additional information and an error icon to identify rules with {ml} jobs that aren't running.
+You can now query data in remote clusters and cold-tier data for index patterns specified in indicator match rules.
 
-*New prebuilt rules*
+*Enhancements to rule preview*
 
-18 new {security-guide}/prebuilt-rules.html[prebuilt rules] were added in 8.6.0.
+You can now risize the {security-guide}/rules-ui-create.html#preview-rules[Rule preview] panel.
 
-[float]
-== Role-based access control (RBAC) for response actions
+*New Torq connector for rule actions*
 
-New feature privileges in {kib} allow admins to manage {security-guide}/endpoint-management-req.html[role-based access to {elastic-defend} features], including access to response actions history, isolating and releasing hosts, and performing process-related response actions on hosts. 
+There's a new Torq connector you can select when configuring rule actions. You must have at least a https://www.elastic.co/pricing[Platinum subscription] to use this connector.
 
-[role="screenshot"]
-image::whats-new/images/8.6/rbac.png[Role-based access controls]
+**Exceptions enhancements**
+
+There are several enhancements and UI improvements to rule exceptions:
+
+* When {security-guide}/add-exceptions.html#detection-rule-exceptions[adding a rule exception], you can set an expiration date for the exception so it only temporarily applies. You can view active and expired exceptions under the *Rule exceptions* tab and choose whether to include expired exceptions when exporting rules.
+* If a field has mapping conflicts, it has a warning icon (image:images/field-warning-icon.png[Field conflict warning icon,13,13]) and {security-guide}/ts-detection-rules.html#rule-exceptions-field-conflicts[message] that describes the conflicts. You can use this information to prevent exception errors and resolve mapping problems.
+* The *Manage rules* button has been renamed to *Link rules*, and the *Import list* button has been renamed to *Import value list*.
+
+*{ml-cap} job enhancements*
+
+{ml-cap} job names are used throughout the {security-app} instead of job IDs so you can quickly know which {ml-cap} jobs you're interacting with.
```

Comment:
```suggestion
{ml-cap} job names now appear instead of job IDs throughout the {security-app} so you can quickly know which {ml-cap} jobs you're interacting with.
```